services/visual_recognition/v3.js

```diff
@@ -14,46 +14,21 @@
  * limitations under the License.
  **/
 
-// Watson Visual Recognition functions supported by this node
-/*var FEATURE_RESPONSES = {
-  classifyImage: 'classifyImage',
-  detectFaces: 'detectFaces',
-  recognizeText: 'recognizeText',
-  createClassifier: 'createClassifier',
-  retrieveClassifiersList: 'retrieveClassifiersList',
-  retrieveClassifierDetails: 'retrieveClassifierDetails',
-  deleteClassifier: 'deleteClassifier'
-};*/
-
 module.exports = function (RED) {
-  var cfenv = require('cfenv');
-  var watson = require('watson-developer-cloud');
-
-  var imageType = require('image-type');
-  var url = require('url');
-  var temp = require('temp');
-  var fileType = require('file-type');
-  var fs = require('fs');
-  var async = require('async');
+  var cfenv = require('cfenv'),
+    watson = require('watson-developer-cloud'),
+    imageType = require('image-type'),
+    url = require('url'),
+    temp = require('temp'),
+    fileType = require('file-type'),
+    fs = require('fs'),
+    async = require('async'),
+    sAPIKey=null, service=null;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/node-red-nodes-watson/node-red-node-watson/file/2737057949/issues/source?bid=3225115&fileBranchId=3352493#l26)